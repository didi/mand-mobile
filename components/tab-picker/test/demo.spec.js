import Demo0 from './cases/demo0'
import Demo1 from './cases/demo1'
import {renderToString} from '@vue/server-test-utils'

describe('TabPicker - Demo', () => {
  test(`Basic`, () => {
    expect(renderToString(Demo0)).toMatchSnapshot()
  })
  test(`TabPicker - quick-block`, () => {
    expect(renderToString(Demo1)).toMatchSnapshot()
  })
})
