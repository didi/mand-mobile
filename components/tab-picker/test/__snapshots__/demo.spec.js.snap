// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabPicker - Demo Basic 1`] = `
<div class="md-example-child md-example-child-tab-picker md-example-child-tab-picker-0">
  <div class="md-tab-picker">
    <div class="md-popup with-mask bottom" style="display:none;">
      <div name="md-mask-fade" class="md-popup-mask" style="display:none;"></div>
      <div name="md-slide-up" class="md-popup-box md-slide-up" style="display:none;">
        <div class="md-popup-title-bar title-align-center large">
          <!---->
          <div class="title-bar-title">
            <p class="title">请选择</p>
            <p class="describe">请选择您所在的省份、城市、区县</p>
          </div>
          <!---->
          <div class="title-bar-right md-popup-close"><i class="md-icon icon-font md-icon-close close lg" style="color:;"></i></div>
        </div>
        <div class="md-tab-picker-content">
          <div class="md-tabs">
            <nav class="md-tab-bar">
              <div class="md-tab-bar-inner">
                <!---->
                <div class="md-scroll-view">
                  <!---->
                  <div scroll-wrapper="" class="scroll-view-container">
                    <!---->
                    <div class="md-tab-bar-list" style="width:0px;"></div> <span class="md-tab-bar-ink" style="width:0px;transform:translateX(0px);"></span>
                    <!---->
                  </div>
                  <!---->
                </div>
              </div>
            </nav>
            <div class="md-tabs-content">
              <div class="md-scroll-view">
                <!---->
                <div scroll-wrapper="" class="scroll-view-container">
                  <!---->
                  <div role="tabpanel" tab="province" name="md-tab-slide-left" class="md-tab-pane">
                    <div class="md-radio-list">
                      <div class="md-cell-item md-radio-item">
                        <div class="md-cell-item-body">
                          <!---->
                          <div class="md-cell-item-content">
                            <p class="md-cell-item-title">北京市</p>
                            <!---->
                          </div>
                          <div class="md-cell-item-right"><label class="md-radio">
                              <div class="md-radio-icon">
                                <!---->
                              </div>
                              <!---->
                            </label>
                            <!---->
                          </div>
                        </div>
                        <!---->
                      </div>
                      <div class="md-cell-item md-radio-item no-border">
                        <div class="md-cell-item-body">
                          <!---->
                          <div class="md-cell-item-content">
                            <p class="md-cell-item-title">四川省</p>
                            <!---->
                          </div>
                          <div class="md-cell-item-right"><label class="md-radio">
                              <div class="md-radio-icon">
                                <!---->
                              </div>
                              <!---->
                            </label>
                            <!---->
                          </div>
                        </div>
                        <!---->
                      </div>
                      <!---->
                    </div>
                  </div>
                  <!---->
                </div>
                <!---->
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
